### PR TITLE
docs: fix node and react usage examples

### LINKED ISSUE #68: Usage example cannot be used as is
#### Describe the bug
The [usage example for node](https://github.com/asyncapi/cupid#usage) cannot be used as is, as the generation of the relationships is asynchronous and does not return the result but a promise.

I had to adapt it to:

```js
const cupid = require('@asyncapi/cupid');
const path = require('path');
const fs = require('fs');

async function getAsyncApiExamples() {
  const docs = [];
  const files = [
    ..
  ]
  for (const file of files) {
    const asyncApiDoc = fs.readFileSync(file, 'utf8');
    docs.push(asyncApiDoc);
  }
  try {
    const mermaidFlowchart = await cupid.getRelations(docs,{syntax:'mermaid'});
    console.log(mermaidFlowchart);
  } catch (error) {
    console.error(error);
  }
}
getAsyncApiExamples();

```

#### How to Reproduce
Use the example code as-is.

#### Expected behavior
That the example works as is without any changes.

### PR DESCRIPTION
**Pull request by arjungarg07**
**Description**
This PR updates the node usage example of cupid which can be used as is.

**Related issue(s)**

Resolves #68

### CODE REVIEW

**Review by derberg — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -63,18 +73,23 @@ const cupid = require('@asyncapi/cupid');
...
 const fs = require('fs');
 
-const examplesPath = './test/examples/flightService';
-
-function getAsyncApiExamples() {
+async function getAsyncApiExamples() {
   const docs = [];
-  const files = fs.readdirSync(examplesPath);
+  const files = [
+    ..
```
Comment (magicmatatjahu) [inline README.md:79]:
```suggestion
    ...
```

Usually for that are using three dots, not two 😅

File: README.md [hunk 2]
```diff
@@ -221,9 +236,9 @@ Based on FlightService example.
 import React from 'react';
 import ReactFlow from 'react-flow-renderer';
 import cupid from '@asyncapi/cupid';
-import {getAsyncApiDocuments} from 'utils'; // function for reading AysncAPI files 
+import {getAsyncApiExamples} from 'utils'; // function for reading AsyncAPI files
```
Comment (magicmatatjahu) [inline README.md:239]:
```suggestion
import {getAsyncApiExamples} from './utils'; // function for reading AsyncAPI files 
```

in some Node version (in your case) `utils` indicates global path, located in folder where node is installed, so please change it to the local path.

**Review by magicmatatjahu — CHANGES_REQUESTED** [review]
Only little improvements, overall good :)

**Review by magicmatatjahu — APPROVED** [review]
LGTM

**Pull request merged**